### PR TITLE
webui: Updated remaining display options where we do not show sysName if hostname is IP

### PR DESCRIPTION
**Pull request by laf**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

Fixes: #4715 

Think this catches the last of the hostnames that are IPs but ip_to_sysname option is on. The name of the function may not be the best choice - any suggestions welcome.

### CODE REVIEW

**Comment by murrant** [comment]
How about display_device_name() or format_hostname()

File: includes/common.php [hunk 1]
```diff
@@ -1128,21 +1126,21 @@ function inet6_ntop($ip)
...
 /**
- * Convert IP to use sysName
+ * If hostname is an ip, use return sysName
  * @param array device
- * @param string ip address
+ * @param string hostname
  * @return string
 **/
-function ip_to_sysname($device, $ip)
+function process_hostname($device, $hostname)
```
Comment (murrant) [inline includes/common.php]: You could make the second argument optional, that would make most of the calls cleaner.

**Comment by aldemira** [comment]
Last time I've forced ip_to_sysname, the devices which didn't have any sysname became unclickable in the all devices page. Maybe I should put this as an issue.

**Comment by laf** [comment]
@murrant all updated

@aldemira also taken care of in this, we only use sysName if it's available.

**Pull request merged**